modules/deploy.nix

```diff
@@ -214,6 +214,8 @@ in {
     # TODO: Handle signals to kill the async command
     nixus.pkgs.writeScript "deploy" ''
       #!${nixus.pkgs.runtimeShell}
+      trap "echo Received signal, exiting; exit" INT TERM
+      trap "kill 0" EXIT
```

Comment:
Looking at this again, I'm not actually sure how this does anything. According to [this answer](https://unix.stackexchange.com/a/169899/214651), `kill -0` doesn't really do anything.

Reply:
Hm...

Reply:
FWIW, this is `kill 0`, not `kill -0` -- 0 is the PID (special-cased?).

Reply:
Ahhh, yeah, `kill 0` is documented as
```
0      All processes in the current process group are signaled.
```

Makes sense now